### PR TITLE
Rkhan/vulidate verbiage

### PR DESCRIPTION
**Pull request by RizaHKhan**
#7

### CODE REVIEW

**Comment by RizaHKhan** [comment]
@JadRizk I am not very proficient with Typescript yet, so I wanted to point out that use of the `any` type might not be to your liking. 

Used in the vuelidateRules.ts file.

**Review by JadRizk — CHANGES_REQUESTED** [review]

File: src/services/vuelidateRules.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { required, minLength, email, helpers } from '@vuelidate/validators'
```
[the remark this answers is not in the document]
Comment (RizaHKhan) [reply src/services/vuelidateRules.ts]: That sounds better, adjusted.

File: src/services/vuelidateRules.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const rules = {
+  userEmail: {
+    email: helpers.withMessage((v: any) => `Please enter a valid email`, email),
```
[the remark this answers is not in the document]
Comment (RizaHKhan) [reply src/services/vuelidateRules.ts]: Adjustment made

File: src/services/vuelidateRules.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    required,
+  },
+  userName: {
+    minLength: helpers.withMessage(
+      (v: any) => `Please enter at least ${v.$params.min} characters`,
+      minLength(4),
+    ),
+    required,
+  },
+}
```
[the remark this answers is not in the document]
Comment (RizaHKhan) [reply src/services/vuelidateRules.ts]: adjustment made, definitely better.

**Comment by RizaHKhan** [comment]
@JadRizk I made an adjustment to the login form to use the email and not the username. 
That seems to be standard convention and some of the logic was already referencing email.

**Review by JadRizk — APPROVED** [review]

**Pull request merged**